packages/dataviews/src/normalize-fields.ts

```diff
@@ -18,7 +18,7 @@ export function normalizeFields< Item >(
 
 		const getValue =
 			field.getValue ||
-			( ( { item }: { item: ItemRecord } ) => item[ field.id ] );
+			( ( { item }: { item: Item } ) => item[ field.id as keyof Item ] );
```

Comment:
Why did you change this?

Reply:
Sorry I didn't notice this before. I see this was addressed at https://github.com/WordPress/gutenberg/pull/64367

Reply:
Yeah, not important right now :)